<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>approverpage</class>
 <widget class="QMainWindow" name="approverpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#approverpage{
	border-image: url(:/picture5.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="cardTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchid">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="placeholderText">
     <string>Search the recordid...</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="controlPage_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_approvealter">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Approve!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Approver Alter File</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>279</width>
        <height>205</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>recordid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="choooserecordid">
           <property name="placeholderText">
            <string>Sure recordid?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>fileid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="thefileid">
           <property name="placeholderText">
            <string>Sure fileid?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>changewhy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="changeforwhat">
           <property name="placeholderText">
            <string>change for what</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>changedate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="thechangedate">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>  the date</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>approverid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="approverownid">
           <property name="placeholderText">
            <string>approver own id</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_logoutapprover">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_count_approve">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowcount_approve">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>cardTable</zorder>
   <zorder>searchid</zorder>
   <zorder>controlPage_3</zorder>
   <zorder>pushButton_logoutapprover</zorder>
   <zorder>pushButton_count_approve</zorder>
   <zorder>rowcount_approve</zorder>
   <zorder>pushButton_search</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
